### DISCUSSION TITLE
How to use extensions with the CLI?

### DISCUSSION THREAD

**Discussion by y76** [GoogleChrome/lighthouse#14848] [General]
Say I want to run the lighthouse CLI with an extension enabled and once without, how would I go about doing that? I can't find any useful information on this. And if I can't do this on the CLI, are there any code examples for the node module?

without extension

lighthouse <URL> 

with extension
lighthouse ??? <URL>

**Comment by adamraine** [comment]
https://github.com/GoogleChrome/lighthouse/blob/main/docs/plugins.md

Short answer:

```sh
lighthouse --plugins=lighthouse-plugin-example
```
